travis-ymls/dask-ml.travis.yml

```diff
@@ -0,0 +1,75 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : dask-ml
+# Source Repo         : https://github.com/dask/dask-ml
+# Travis Job Link     : https://travis-ci.com/github/asellappen/dask-ml/
+# Created travis.yml  : Yes
+# Maintainer          : Arumugam N S <asellappen@yahoo.com> / Priya Seth<sethp@us.ibm.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+language: python
+
+dist: focal
+
+matrix:
+  include:
+  - env: 
+     - PYTHON_VERSION=3.6
+     - envFile='ci/environment-3.6.yaml'
+  - env: $PYTHON_VERSION=3.7
+     - PYTHON_VERSION=3.7
```

Comment:
Are both these variables (with and without $) required ?

Reply:
no ,not required. corrected now.